charts/lotus-bundle/README.md

```diff
@@ -0,0 +1,22 @@
+# Lotus Bundle
+
+If you are writing a lotus application, bundle your application with a lotus node.
+
+
+## Default behaviors:
+
+Lotus will run as a sidecar to your application. You can access lotus at 127.0.0.1:1234.
+There is no authentication, and lotus will not be accessible outside of your pod.
```

Comment:
This might be as simple as

```
  volumes:
  - name: lotus-path
    mountPath: /var/lib/lotus
    readOnly: true
    
  ...
  env:  
  - name: LOTUS_PATH
    value: /var/lib/lotus
```

This would require that the backing volume support multiple readers and I don't remember if this is common or not.

Reply:
I want to say that because they are in the same pod this isn't an issue, and should just work as long as there is only one writer.

Reply:
You're right. Adding the lotus-path into the application container as read-only.